app/components/product/view.hbs

```diff
@@ -78,10 +78,17 @@
             @subtitle="Informatie over de leverancier waar het product aangekocht is." />
           <div class="mt-6 grid grid-cols-1 gap-y-6 gap-x-4 grid-cols-6">
             <div class="col-span-4 sm:col-span-3">
-              <Util::Property @label="Leverancier" @value={{@model.purchaseOffering.businessEntity.name}} />
+              <Util::Property @label="Leverancier">
+                {{@model.purchaseOffering.businessEntity.name}}
+                <Rlv::Button
+                  @skin="link"
+                  @href={{@model.purchaseOffering.businessEntity.rkbLink}}
+                  @icon="external-link-line"
+                />
```

Comment:
Identifier (and thus link to RKB) may be empty. Best to make showing the icon conditional. Also holds for other places in the interface.